README.md

```diff
@@ -1,64 +1,199 @@
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
-
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+---
+services: active-directory
+platforms: java
+author: ramya25
+level: 200
+client: java console app
+service: Microsoft Graph
+endpoint: Microsoft identity platform
+page_type: sample
+languages:
+  - java  
+products:
+  - azure
+  - azure-active-directory  
+  - office-ms-graph
+description: "This sample demonstrates a java console application calling The Microsoft Graph"
+---
+# Invoking an API protected by Azure AD from a text-only device
+
+## About this sample
+
+### Overview
+
+This sample uses the OAuth2 **device code flow** and demonstrates how to leverage MSAL4J from apps that **do not have the capability of offering an interactive authentication experience**. It enables these apps to:
+
+- Authenticate a user
+- And call to a web API (in this case, the [Microsoft Graph](https://graph.microsoft.com))
+
+To emulate a device not capable of showing UX, the sample is packaged as a Java console application.
+The application signs-in users with Azure Active Directory (Azure AD), using the Microsoft Authentication Library for Java (MSAL4J) to obtain a JWT access token through the OAuth 2.0 protocol. The access token is then used to call the Microsoft Graph API to obtain information about the user who signed-in. The sample is structured so that you can call your own API
+
+![Topology](./ReadmeFiles/Topology.png)
+
+### Scenario
+
+The application obtains tokens through a two steps process especially designed for devices and operating systems that cannot display any UX. Examples of such applications are applications running on iOT, or Command-Line tools (CLI). The idea is that:
+
+1. whenever a user authentication is required, the command-line app provides a code and asks the user to use another device (such as an internet-connected smartphone) to navigate to [https://microsoft.com/devicelogin](https://microsoft.com/devicelogin), where the user will be prompted to enter the code. That done, the web page will lead the user through a normal authentication experience, including consent prompts and multi factor authentication if necessary.
```

Comment:
typo: uppercase W in Whenever
reword: Once that is done, the web page ....

Reply:
Updated